src/main/webapp/app/code-editor/code-editor.module.ts

```diff
@@ -49,9 +48,9 @@ const ENTITY_STATES = [...codeEditorRoute];
         ArtemisSharedModule,
         AceEditorModule,
         ArtemisResultModule,
-        ArtemisMarkdownEditorModule,
         MomentModule,
-        ArtemisProgrammingExerciseModule,
+        ArtemisProgrammingExerciseInstructionsEditorModule,
```

Comment:
Because of the restructuring, the code-editor now doesn't have to import the whole programming-exercise.module